lib/elixir/test/elixir/exception_test.exs

```diff
@@ -701,33 +701,65 @@ defmodule ExceptionTest do
       message = blame_message(%{first: nil, second: nil}, fn map -> map.firts end)
 
       assert message == """
-             key :firts not found in: %{first: nil, second: nil}. Did you mean:
+             key :firts not found in:
+
+                 %{first: nil, second: nil}. Did you mean:
 
                    * :first
              """
 
       message = blame_message(%{"first" => nil, "second" => nil}, fn map -> map.firts end)
 
-      assert message == "key :firts not found in: %{\"first\" => nil, \"second\" => nil}"
+      assert message == """
+             key :firts not found in:
+
+                 %{"first" => nil, "second" => nil}\
+             """
 
       message =
         blame_message(%{"first" => nil, "second" => nil}, fn map -> Map.fetch!(map, "firts") end)
 
-      assert message == "key \"firts\" not found in: %{\"first\" => nil, \"second\" => nil}"
+      assert message ==
+               """
+               key "firts" not found in:
+
+                   %{"first" => nil, "second" => nil}\
+               """
 
       message =
-        blame_message([first: nil, second: nil], fn kwlist -> Keyword.fetch!(kwlist, :firts) end)
+        blame_message(
+          [
+            created_at: nil,
+            updated_at: nil,
+            deleted_at: nil,
+            started_at: nil,
+            finished_at: nil
+          ],
+          fn kwlist ->
+            Keyword.fetch!(kwlist, :inserted_at)
+          end
+        )
 
       assert message == """
-             key :firts not found in: [first: nil, second: nil]. Did you mean:
-
-                   * :first
+             key :inserted_at not found in:
+
+                 [
+                   created_at: nil,
+                   updated_at: nil,
+                   deleted_at: nil,
+                   started_at: nil,
+                   finished_at: nil
+                 ]. Did you mean:
+
+                   * :created_at
+                   * :finished_at
+                   * :started_at
```

Comment:
This is also an issues:
I think it should be something like:
```
key :firts not found, did you mean:

    * :first

in:

    [ data: :structure]
...
```
This is more natural imo - you first check the suggestion and then go to the printed structure.
With the current implementation, you have to go back after reading the suggestion.

Reply:
Let's not change the order for now but we definitely need to fix the line breaking.